src/app/dao/tests/ChatDAO.spec.ts

```diff
@@ -19,7 +19,7 @@ import { UserDAO } from '../UserDAO';
 import { ChatService } from 'src/app/services/ChatService';
 import { Subscription } from 'rxjs';
 
-describe('ChatDAO', () => {
+xdescribe('ChatDAO', () => {
```

Comment:
nooon

Reply:
faut qu'on interdise le xdescribe dans le linter car à un moment tu vas faire passer ça :p

Reply:
c'est déjà le cas non ? En tout cas le fdescribe ça fait des warning !

Reply:
mais warning is not forbidden !
Seulement les erreurs. J'ajoute au ticket linter et tpr

Reply:
j'unresolve car tu as réintroduit ceci !

Reply:
j'ai résolu !